docs/toc.yml

```diff
@@ -778,9 +781,6 @@ items:
       href: outlook/outlook-addin-design.md
     - name: Event-based activation
       items:
-      - name: Implement an event-based add-in
```

Comment:
Can we keep this entry to complete the event-based activation story for Outlook in the TOC? However, if we deem it redundant, I'm okay with removing it.